app/models/spree/variant_decorator.rb

```diff
@@ -12,7 +12,7 @@
 
   has_many :display_images, through: :display_variant_images, source: :image
 
-  alias_method :images, :variant_image_images
   alias_method :images=, :variant_image_images=
+  alias_method :images, :variant_image_images
```

Comment:
Consider this a picked nit, but does this order matter?

Reply:
I guess not. I'll back out this specific chunk.